tensorflow_probability/python/stats/quantiles.py

```diff
@@ -548,23 +548,20 @@ def percentile(x,
       axis = _make_static_axis_non_negative_list(axis, x_ndims)
       y = _move_dims_to_flat_end(x, axis, x_ndims, right_end=True)
 
-    frac_at_q_or_above = 1. - q / 100.
+    frac_at_q_or_above = q / 100.
 
-    # Sort everything, not just the top 'k' entries, which allows multiple calls
-    # to sort only once (under the hood) and use CSE.
+    # Sort (in ascending order) everything, not just the top 'k' entries, which
```

Comment:
you can delete `not just the top k entries`, since that was referring to the `_sort_tensor` implementation.

Reply:
Done.